public/src/js/controllers/transactions.js

```diff
@@ -109,13 +109,79 @@ function($scope, $rootScope, $routeParams, $location, Global, Transaction, Trans
     });
   };
 
+  var _sidechainId = function (transaction) {
+    // This is the case for: Sidechain creation TX, version: -4
+    // https://explorer-testnet.horizen.io/api/tx/c205cc95d71dd7f008fe8691a79781b9d0a25ddfa2d34f367839d73b02b3e3a5
+
+    if (transaction.vsc_ccout && transaction.vsc_ccout.length > 0) {
+      return transaction.vsc_ccout[0]["scid"]
+    }
+
+    // This is the case for: MC -> SC Forward Transfer TX, version: -4
+    // https://explorer-testnet.horizen.io/api/tx/e07a3bcbdfad4b022a3f2802bb57f6445abf82daee7f076b6e39f3f6406e9848
+
+    if (transaction.vft_ccout && transaction.vft_ccout.length > 0) {
+      return transaction.vft_ccout[0]["scid"]
+    }
+
```

Comment:
We could refactor this a little bit, we could create two booleans: 
```
const isSidechainCreationTransaction = transaction.vsc_ccout && transaction.vsc_ccout.length > 0
const isForwardTransferTransaction = transaction.vft_ccout && transaction.vft_ccout.length > 0

if (isSidechainCreationTransaction || isForwardTransferTransaction) {
    return transaction.vft_ccout[0]["scid"]
}

```

Reply:
You're right. I left it this way to have the cases well separated (and commented) but if you prefer, I can optimize it

Reply:
Leave as it is, there are not so many transaction types